src/compile/facet.ts

```diff
@@ -136,6 +136,7 @@ export class FacetModel extends ModelWithField {
   public parseMark() {
     this.child.parseMark();
 
+    const facetByTwo = this.channelHasField(ROW) && this.channelHasField(COLUMN);
```

Comment:
Yeah -- but in any case both `facetByTwo` and  `crossFacet`are terrible name -- better figure a better one.